src/Util/Str.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace GrumPHP\Util;
+
+class Str
+{
+    public static function contains($haystack, $needles)
```

Comment:
Could you rename this method to: containsOneOf()?
Can you add strict parameter types + return type.
I prefer to enforce the needles to be an `array` of strings - which makes more sense given the new name.

Reply:
One concern with the method name .. Not only can needles be a array of strings, but it can also be just a single string ...

So for me it would read weird in this use case:
```php
Str::containsOneOf('This string contains this word', 'word');
```

Reply:
I re-read what you typed and i think you are suggesting to `only` allow needles to be an `array` of strings which then would accommodate the new method name ...

I will make these changes and push shortly.

Reply:
This should resolve this conversation https://github.com/phpro/grumphp/pull/541/commits/2de742761c1857f9e3a8a79363f215b75bfb428b